src/types/mod.rs

```diff
@@ -1792,6 +1792,33 @@ pub struct ChainParametersV1 {
     pub pool_parameters:          PoolParameters,
 }
 
+#[derive(common::Serial, Debug)]
+/// Values of chain parameters that can be updated via chain updates.
+pub struct ChainParametersV2 {
+    /// Consensus protocol version 2 timeout parameters.
+    pub timeout_parameters:                TimeoutParameters,
+    /// Minimum time interval between blocks.
+    pub min_block_time:                    SlotDuration,
```

Comment:
Is there a reason why we don't just use Duration here?

Reply:
Right, when introducing this type, Duration did not implement `concordium_base::Serial`.
I'll change this